diff -x config.log -x config.status -ru work/gnutls-3.3.1/guile/src/core.c work/gnutls-3.3.1/guile/src/core.c
--- work/gnutls-3.3.1/guile/src/core.c	2013-10-10 12:10:49.000000000 +0100
+++ work/gnutls-3.3.1/guile/src/core.c	2014-04-22 12:46:55.558190932 +0100
@@ -3366,7 +3366,6 @@
   /* Use Guile's allocation routines, which will run the GC if need be.  */
   gnutls_malloc = scm_malloc;
   gnutls_realloc = scm_realloc;
-  gnutls_secure_malloc = scm_malloc;
   gnutls_free = free;
 
   (void) gnutls_global_init ();
