diff -x config.log -x config.status -ru work/spell-1.1/Makefile.in work/spell-1.1/Makefile.in
--- work/spell-1.1/Makefile.in	2010-05-04 21:04:35.000000000 +0200
+++ work/spell-1.1/Makefile.in	2012-11-21 12:57:38.982069647 +0100
@@ -18,7 +18,7 @@
 
 
 # Start of system configuration section.
-
+DESTDIR =
 SHELL = /bin/sh
 VPATH = @srcdir@
 srcdir = @srcdir@
@@ -66,10 +66,10 @@
 	$(CC) $(LDFLAGS) $(OBJS) $(LIBS) -o $@
 
 install: installdirs install-info
-	$(INSTALL_PROGRAM) spell $(bindir)/spell
+	$(INSTALL_PROGRAM) spell $(DESTDIR)$(bindir)/spell
 
 install-info:
-	$(INSTALL_DATA) $(srcdir)/spell.info $(infodir)/spell.info
+	$(INSTALL_DATA) $(srcdir)/spell.info $(DESTDIR)$(infodir)/spell.info
 
 uninstall:
 	rm -f $(bindir)/spell $(infodir)/spell.info
@@ -137,7 +137,7 @@
 installcheck:
 
 installdirs: mkinstalldirs
-	$(srcdir)/mkinstalldirs $(bindir) $(infodir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(infodir)
 
 Makefile: Makefile.in config.status
 	CONFIG_FILES=$@ CONFIG_HEADERS= ./config.status
