diff -x config.log -x config.status -ru work/gcl/gcl-tk/tkMain.c work/gcl/gcl-tk/tkMain.c
--- work/gcl/gcl-tk/tkMain.c	2013-08-07 00:53:26.000000000 +0200
+++ work/gcl/gcl-tk/tkMain.c	2013-10-04 11:31:26.851131864 +0200
@@ -31,6 +31,10 @@
 /*  static char rcsid[] = "$Header$ SPRITE (Berkeley)"; */
 /*  #endif */
 
+#ifndef USE_INTERP_RESULT
+#define USE_INTERP_RESULT
+#endif
+
 #include <string.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -43,7 +47,6 @@
 #define TkCreateMainWindow Tk_CreateMainWindow
 #endif
 
-
 /*-------------------------------------------------------------------*/
 #include <unistd.h>
 #include <sys/types.h>
