diff -x config.log -x config.status -ru tmp/slib-3b4/Makefile work/slib-3b4/Makefile
--- tmp/slib-3b4/Makefile	2013-06-07 15:54:56.000000000 +0200
+++ work/slib-3b4/Makefile	2013-08-17 13:49:05.811669809 +0200
@@ -284,7 +284,7 @@
 uninstall48:
 	-rm $(DESTDIR)$(S48LIB)$(IMAGE48)
 
-install-lib: $(libfiles) installdirs
+install-lib: installdirs
 	-$(INSTALL_DATA) $(libfiles) $(DESTDIR)$(libslibdir)
 
 install-script: slib.sh installdirs
@@ -297,7 +297,7 @@
 	$(INSTALL_PROGRAM) slib-script $(DESTDIR)$(bindir)slib
 	rm slib-script
 
-install: install-script install-lib install-infoz install-man
+install: install-script install-lib #install-infoz install-man
 
 uninstall: uninstall48
 	$(PRE_UNINSTALL)     # Pre-uninstall commands follow.
diff -x config.log -x config.status -ru tmp/slib-3b4/slib.texi work/slib-3b4/slib.texi
--- tmp/slib-3b4/slib.texi	2013-06-07 15:54:56.000000000 +0200
+++ work/slib-3b4/slib.texi	2013-08-17 12:42:23.872664771 +0200
@@ -813,10 +813,10 @@
 almost all the information neccessary for submitting a problem report.
 An unspecified value is returned.
 
-@defunx slib:report #t
+@defun slib:report #t
 provides a more verbose listing.
 
-@defunx slib:report filename
+@defun slib:report filename
 Writes the verbose report to file @file{filename}.
 
 @example
