diff -x config.log -x config.status -ru work/gnuspool-1.8/Makefile.in work/gnuspool-1.8/Makefile.in
--- work/gnuspool-1.8/Makefile.in	2010-01-30 19:52:13.000000000 +0100
+++ work/gnuspool-1.8/Makefile.in	2012-11-10 12:20:23.774584437 +0100
@@ -63,8 +63,8 @@
 	cp -r -p ptrinstall $(DATADIR)
 	cd doc;$(MAKE) install
 	cd cupspy;$(MAKE) install
-	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -R
-	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -p ALL `who am i|sed -e 's/ .*//'`
+#	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -R
+#	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -p ALL `who am i|sed -e 's/ .*//'`
 	cd udev;$(MAKE) install
 
 install-strip:	all
@@ -77,8 +77,8 @@
 	cp -r -p ptrinstall $(DATADIR)
 	cd doc;$(MAKE) install-strip
 	cd cupspy;$(MAKE) install-strip
-	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -R
-	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -p ALL `who am i|sed -e 's/ .*//'`
+#	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -R
+#	$(STAGED_INSTALL) || $(BINDIR)/gspl-uchange -p ALL `who am i|sed -e 's/ .*//'`
 	cd udev;$(MAKE) install
 
 install-user user-install:
diff -x config.log -x config.status -ru work/gnuspool-1.8/doc/Makefile.in work/gnuspool-1.8/doc/Makefile.in
--- work/gnuspool-1.8/doc/Makefile.in	2010-07-01 21:55:35.000000000 +0200
+++ work/gnuspool-1.8/doc/Makefile.in	2012-11-10 12:20:23.791251109 +0100
@@ -40,6 +40,11 @@
 	# Currently don't do anything
 
 install install-strip: all
+	$(INSTALL) -d -m 755 $(MANDIR)/man1
+	$(INSTALL) -d -m 755 $(MANDIR)/man3
+	$(INSTALL) -d -m 755 $(MANDIR)/man5
+	$(INSTALL) -d -m 755 $(MANDIR)/man8
+	$(INSTALL) -d -m 755 $(PDFDIR)
 	$(INSTALL_DATA) poddoc/man/*.1 $(MANDIR)/man1
 	$(INSTALL_DATA) poddoc/man/*.3 $(MANDIR)/man3
 	$(INSTALL_DATA) poddoc/man/*.5 $(MANDIR)/man5
diff -x config.log -x config.status -ru work/gnuspool-1.8/src/apilib/Makefile.in work/gnuspool-1.8/src/apilib/Makefile.in
--- work/gnuspool-1.8/src/apilib/Makefile.in	2010-02-01 16:59:36.000000000 +0100
+++ work/gnuspool-1.8/src/apilib/Makefile.in	2012-11-10 12:20:23.791251109 +0100
@@ -73,11 +73,15 @@
 	sh ./Hdrmake.sh $(BASE)/config.h $(HDRS)
 
 install: all
+	$(INSTALL) -d -m 755 $(INCLUDEDIR)
+	$(INSTALL) -d -m 755 $(LIBDIR)
 	libtool --mode=install $(INSTALL) $(APILIB) $(LIBDIR)
 	$(INSTALL_DATA) gspool.h $(INCLUDEDIR)
 	$(INSTALL_DATA) $(APILIB) $(LIBDIR)
 
 install-strip: all
+	$(INSTALL) -d -m 755 $(INCLUDEDIR)
+	$(INSTALL) -d -m 755 $(LIBDIR)
 	libtool --mode=install $(INSTALL) -s $(APILIB) $(LIBDIR)/$(APILIB)
 	$(INSTALL_DATA) gspool.h $(INCLUDEDIR)
 	$(INSTALL_DATA) $(APILIB) $(LIBDIR)
diff -x config.log -x config.status -ru work/gnuspool-1.8/src/lib/Makefile.in work/gnuspool-1.8/src/lib/Makefile.in
--- work/gnuspool-1.8/src/lib/Makefile.in	2010-10-04 13:12:51.000000000 +0200
+++ work/gnuspool-1.8/src/lib/Makefile.in	2012-11-10 12:20:23.791251109 +0100
@@ -71,12 +71,14 @@
 	libtool --mode=link @CC@ $(CCFLAGS) -o $(LIBCURS) $(LIBCURSOBJS) $(LIB) $(LIBCURSES) -rpath $(RPATH) -version-info 0:0:0
 
 install: all
+	$(INSTALL) -d -m 755 $(LIBDIR)
 	libtool --mode=install $(INSTALL) $(LIB) $(LIBDIR)
 	libtool --mode=install $(INSTALL) $(LIBCURS) $(LIBDIR)
 	$(RM) $(LIBDIR)/libgnuspool_int.a $(LIBDIR)/libgnuspool_curs.a 
 	$(INSTALL_DATA) $(LIB) $(LIBCURS) $(LIBDIR)
 
 install-strip: all
+	$(INSTALL) -d -m 755 $(LIBDIR)
 	libtool --mode=install $(INSTALL) -s $(LIB) $(LIBDIR)
 	libtool --mode=install $(INSTALL) -s $(LIBCURS) $(LIBDIR)
 	$(RM) $(LIBDIR)/libgnuspool_int.a $(LIBDIR)/libgnuspool_curs.a 
diff -x config.log -x config.status -ru work/gnuspool-1.8/src/lpdint/Makefile.in work/gnuspool-1.8/src/lpdint/Makefile.in
--- work/gnuspool-1.8/src/lpdint/Makefile.in	2009-05-18 13:27:26.000000000 +0200
+++ work/gnuspool-1.8/src/lpdint/Makefile.in	2012-11-10 12:20:23.791251109 +0100
@@ -85,6 +85,7 @@
 	sed -e 's|SPOOLDIR|@spooldir@|' -e 's|DATADIR|@pkgdatadir@|' xtlpc-ctrl >$(DATADIR)/xtlpc-ctrl
 	sed -e 's|SPOOLDIR|@spooldir@|' -e 's|DATADIR|@pkgdatadir@|' xtlpd-ctrl >$(DATADIR)/xtlpd-ctrl
 	chown $(SPOOLUSER) $(DATADIR)/xtlpc-ctrl $(DATADIR)/xtlpd-ctrl
+	$(INSTALL) -d -m 755 $(SYSCONFDIR)
 	echo '1  BSD  -  -' >$(SYSCONFDIR)/gnuspool.ext
 
 install-strip:	all
@@ -96,6 +97,7 @@
 	sed -e 's|SPOOLDIR|@spooldir@|' -e 's|DATADIR|@pkgdatadir@|' xtlpc-ctrl >$(DATADIR)/xtlpc-ctrl
 	sed -e 's|SPOOLDIR|@spooldir@|' -e 's|DATADIR|@pkgdatadir@|' xtlpd-ctrl >$(DATADIR)/xtlpd-ctrl
 	chown $(SPOOLUSER) $(DATADIR)/xtlpc-ctrl $(DATADIR)/xtlpd-ctrl
+	$(INSTALL) -d -m 755 $(SYSCONFDIR)
 	echo '1  BSD  -  -' >$(SYSCONFDIR)/gnuspool.ext
 
 clean:
diff -x config.log -x config.status -ru work/gnuspool-1.8/util/Makefile.in work/gnuspool-1.8/util/Makefile.in
--- work/gnuspool-1.8/util/Makefile.in	2010-07-01 21:33:09.000000000 +0200
+++ work/gnuspool-1.8/util/Makefile.in	2012-11-10 12:20:23.757917764 +0100
@@ -36,7 +36,7 @@
 LEX		=	@LEX@
 LEXLIB		=	@LEXLIB@
 RM		=	rm -f
-PROGS		=	gspl-pmenu gspl-hostedit gspl-xhostedit gspl-passwd
+PROGS		=	gspl-pmenu gspl-hostedit gspl-passwd
 BUILDPROGS	=	helpparse
 BINDIR		=	$(DESTDIR)@bindir@
 SBINDIR		=	$(DESTDIR)@sbindir@
